### PR TITLE
[bn] Localized content/bn/ _index.html

### PR DESCRIPTION
**Pull request by Mouly22**
Signed-off-by: Umme Abira Azmary abiraazmary22@gmail.com
Localizes ```content/bn/_index.html>```  to Bengali
This is an intial PR as may need some improvements!
Tracking issue:

### CODE REVIEW

**Comment by mitul3737** [comment]
@Mouly22 Kindly create a pull req to merge it to `Kubernetes Bengali Documentation: main`

**Comment by mitul3737** [comment]
Great job. You have localized a big content! @Mouly22

**Comment by Mouly22** [comment]
sure!

**Review by mitul3737 — CHANGES_REQUESTED** [review]

**Review by Mouly22 — COMMENTED** [review]
@mitul3737 is it okay now?

File: Kubernetes Bengali Documentation/content/bn/_index.html [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+{{% blocks/feature image="flower" %}}
+[কুবারনেটস]({{< relref "/docs/concepts/overview/what-is-kubernetes" >}}), K8s নামেও পরিচিত, কনটেইনারাইজড অ্যাপ্লিকেশনের স্বয়ংক্রিয় স্থাপনা, স্কেলিং এবং পরিচালনার জন্য একটি ওপেন-সোর্স সিস্টেম।
+
+এটি কন্টেইনারগুলিকে গোষ্ঠীভুক্ত করে যা সহজ ব্যবস্থাপনা এবং আবিষ্কারের জন্য লজিক্যাল ইউনিটে একটি অ্যাপ্লিকেশন তৈরি করে। কুবারনেটস সর্বোত্তম-জাত ধারণা এবং অনুশীলনের সাথে মিলিতভাবে [Google-এ উৎপাদন কাজের লোড চালানোর অভিজ্ঞতা 15 বছরের](http://queue.acm.org/detail.cfm?id=2898444), সম্প্রদায় তৈরি করে।
+{{% /blocks/feature %}}
+
+{{% blocks/feature image="scalable" %}}
+#### বিশ্বব্যাপী স্কেল ক্ষমতা <!--Planet Scale-->
+
+Google সপ্তাহে বিলিয়ন কন্টেইনার চালানোর অনুমতি একই নীতিতে ডিজাইন করে পায়, সেই একই নীতিতে ডিজাইন এটি ডিজাইন করা হয় দেয়, Kubernetes ব্যবহারকারীর অপস টিম না বাড়িয়ে স্কেল করতে পারে।
```
[the remark this answers is not in the document]
Comment (Mouly22) [reply Kubernetes Bengali Documentation/content/bn/_index.html]: 🙂

**Review by mitul3737 — APPROVED** [review]

**Pull request merged**